plugins/offboard/offboard.h

```diff
@@ -94,7 +94,15 @@ class Offboard : public PluginBase {
     };
 
     /**
-     * @brief Start offboard control (synchronous).
+     * @brief Type for position commands in NED coordinates (north, east, down).
+     */
+    struct PositionLocalNED {
+        float x_m; /**< @brief X Position in NED frame in meters.*/
+        float y_m; /**< @brief Y Position in NED frame in meters.*/
+        float z_m; /**< @brief Z Position in NED frame in meters.*/
+    };
```

Comment:
I suggest to reuse the convention naming for NED (which is already used by VelocityLocalNED)
north_m, east_m, down_m